librz/analysis/op.c

```diff
@@ -213,6 +214,8 @@ RZ_API bool rz_analysis_op_ismemref(int t) {
 	case RZ_ANALYSIS_OP_TYPE_STORE:
 	case RZ_ANALYSIS_OP_TYPE_LEA:
 	case RZ_ANALYSIS_OP_TYPE_CMP:
+	case RZ_ANALYSIS_OP_TYPE_POP:
```

Comment:
Nice finding!

Reply:
There are probably more of those little in-precise implementations. IMHO the problem is, we miss an abstract concept of stack frames for static analysis.

It's all spread around some struct members and functions like those (all of which miss documentation, so variables like `RzAnalysisOp->stackptr` are used differently between `x86` and other archs). All of which is somehow baked together in `var.c`.